README.md

```diff
@@ -189,28 +190,156 @@ When Silead touchscreen support was added to ACPI (PC) platforms, the
 integrator did not anticipate any uses cases outside the device vendor's
 ecosystems. For this reason, it is required to pass additional
 device parameters to the driver that are neither contained in the firmware
-nor in the ACPI DSDT. silead_ts handles this in the kernel, using an
-ACPI override/quirk table, while gslx680_ts_acpi expects firmware in a
-different format that contains this information.
+nor in the ACPI DSDT. silead_ts handles this in the kernel, using a DMI
+quirk table. gslx680_ts_acpi expects firmware in its own format which contains
+this information.
 
-Since silead_ts requires a kernel modification (and thus a reboot or kexec)
-whenever parameters are changed, testing is easier with gslx680_ts_acpi.
-There, parameters can be changed in the firmware, then updated by unloading and
-reloading the kernel module.
+silead_ts allows passing the device parameters on the kernel commandline
+to test on devices which are not yet in the DMI quirk table. This is supported
+in kernel 6.10 and later, for older kernels the kernel needs to be patched
+and build [(*)](kernel-recompile-debian.md) to test parameter changes.
```

Comment:
```suggestion
and rebuilt [(*)](kernel-recompile-debian.md) to test parameter changes.
```

Reply:
I think you should also replace the `(*)` with a link description. Clicking `(*)` is hard on a small screen and has other accessibility issues. Something like `[see kernel-recompile-debian.md for Debian-specific instructions]`, or something shorter.